### PR TITLE
Final Code merge for Labs30

### PR DESCRIPTION
**Pull request by JeremySpradlin**
- Updated documentation to include cleaning up comments, etc
- Included Design Doc in Notebooks/
- Removed excess files/code
- Added DB Object classes 'Judge' and 'Case'
- Updated README to include state of project information for next team to take up this repo.
- Updated README Tools section
- Added link to Heroku deployment tutorial

### CODE REVIEW

**Review by fansha1994 — APPROVED** [review]
Heroku tutorial got added to README.md file by Jeremy

File: README.md [hunk 1]
```diff
@@ -7,26 +7,36 @@ An application to assist immigration advocates in winning asylum cases
...
 * Create an API connecting the database to the front end with the result that uploaded case files can be inserted into the database
 
+* After converting the uploaded PDF to text, we will extract relvant entities from the text to store relevant features in a DB to facilitate data recall and analysis
+
+
 ## Tools
 
  * [Pytesseract](https://github.com/madmaze/pytesseract)
  * [FastAPI](https://github.com/tiangolo/fastapi)
  * [AWS Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk/)
```
Comment (serinamarie) [inline README.md]: I'd recommend removing Elastic Beanstalk if you didn't use it!

**Review by JeremySpradlin — COMMENTED** [review]
Removed the Beanstalk listing from Tools.

**Comment by serinamarie** [comment]
> Removed the Beanstalk listing from Tools.

Looking good, then!

**Review by serinamarie — APPROVED** [review]

**Pull request merged**